packages/pebble-shared/svgs/phone-missed.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1000 1000"><path fill-rule="evenodd" d="M948.4 694.6C826.4 574.2 661.6 500 480 500c-181.6 0-346.4 74.2-468.4 194.6A42.4 42.4 0 0 0 0 724.2c0 11.6 4.4 22 11.6 29.5l99.2 103.4a39 39 0 0 0 28.4 12c10.8 0 20.8-4.5 28-11.6a456.9 456.9 0 0 1 106.4-77 41.7 41.7 0 0 0 22.4-37.6V613.8a574.6 574.6 0 0 1 368-.4v129.1c0 16.3 9.2 30.8 22.4 37.5a471.6 471.6 0 0 1 106.8 77 38.5 38.5 0 0 0 28 11.8 39 39 0 0 0 28.4-12.1l99.2-103.4a42.4 42.4 0 0 0-.4-58.7zM216 717.9a534 534 0 0 0-74.8 53l-42.8-44.7c36.4-31.2 76-57.9 118-79.1v70.8h-.4zm603.2 52.5c-24-20-48.8-37.5-75.2-53v-70.7a603 603 0 0 1 118 79.1l-42.8 44.6zM280 268l197.6 205.8 282.8-294.5L704 120 477.6 355.8 336 208.3h104V125H200v250h80V268z" clip-rule="evenodd"/></svg>
```

Comment:
for 
<img width="207" alt="Screenshot 2020-08-14 at 2 31 54 PM" src="https://user-images.githubusercontent.com/14153284/90232895-edf5cd80-de3a-11ea-977e-b51a27045a47.png">
are you using `incoming-call` and `outgoing-call` or adding these 2 too?

Reply:
sales app uses old icons because these new icons are not part of agent stats design

Reply:
On second thought, it's best to add any more new icons in this pr